packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -391,27 +391,23 @@ const FlowsheetResultCell = (props) => {
     ...customProps
   } = props;
   const containerDiv = useRef(null);
-  const [contentWidth, setContentWidth] = useState(null);
   const [numericOverflow, setNumericOverflow] = useState(false);
 
-  useLayoutEffect(() => {
+  useEffect(() => {
     if (!containerDiv.current || !resultDataSet[0]) {
       return;
     }
+    const contentWidth = containerDiv.current.children[0].getBoundingClientRect().width;
+    const containerWidth = containerDiv.current.getBoundingClientRect().width;
```

Comment:
It does not appear that this logic needed to move outside the if block.  By moving, we would be required to calculate and obtain two bounding client rectangles in cases that are unnecessary.

Reply:
Updated in this commit! https://github.com/cerner/terra-clinical/pull/902/commits/7feca5e6dbeee41a971fc1248330b0da352dc0e3#diff-2ef96e32bd2562338959196c9293a261b049ebfc1776e40247b6c8e15014f9b4